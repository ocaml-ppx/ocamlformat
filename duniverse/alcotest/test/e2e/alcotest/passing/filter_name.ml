let () =
  let open Alcotest in
  let id () = () in
  run ~argv:[| ""; "test"; "test-a" |] __FILE__
    [
      ( "test-a",
        [
          test_case "First test case" `Quick id;
          test_case "Second test case" `Quick id;
        ] );
      ( "test-b",
        [
          test_case "Skipped failing test" `Quick (fun () -> invalid_arg "boom");
        ] );
    ]
