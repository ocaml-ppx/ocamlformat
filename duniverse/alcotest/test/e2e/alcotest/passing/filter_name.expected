Testing `test/e2e/alcotest/passing/filter_name.ml'.
This run has ID `<uuid>'.

  [OK]          test-a          0   First test case.
  [OK]          test-a          1   Second test case.
  [SKIP]        test-b          0   Skipped failing test.

Full test results in `<build-context>/_build/_tests/<test-dir>'.
Test Successful in <test-duration>s. 2 tests run.
